crates/paralegal-policy/src/context.rs

```diff
@@ -321,17 +352,18 @@ impl Context {
             })
     }
 
-    /// Return an example pair for a data flow from an source from `from` to a sink
+    /// Return an example pair for a flow from an source from `from` to a sink
     /// in `to` if any exist.
-    pub fn any_data_flows<'a>(
+    pub fn any_flows<'a>(
         &self,
-        ctrl_id: ControllerId,
+        ctrl_id: Option<ControllerId>,
         from: &[&'a DataSource],
-        to: &[&'a DataSink],
-    ) -> Option<(&'a DataSource, &'a DataSink)> {
+        to: &[&'a CallSiteOrDataSink],
+        edge_type: EdgeType,
+    ) -> Option<(&'a DataSource, &'a CallSiteOrDataSink)> {
         from.iter().find_map(|&src| {
             to.iter().find_map(|&sink| {
-                self.data_flows_to(ctrl_id, src, sink)
+                self.flows_to(ctrl_id, src, sink, edge_type.clone())
```

Comment:
And if `EdgeType` id `Copy` you don't nee to `clone` here 😉

Reply:
done